safe-global/web-core
# fix: change favicon per theme

## What it solves

Resolves https://github.com/safe-global/web-core/issues/1123

## How this PR fixes it
Implements a hook to detect the system's preferred color scheme and subscribes to that media query changes

## How to test it
Go to your OS preferences and change the dark/light theme.
Observe the favicon in the browsers tab changing accordingly.

## Screenshots
![Screenshot 2022-12-12 at 15 41 49](https://user-images.githubusercontent.com/32431609/207073676-26224732-4361-4680-810b-4a37d11ba50e.png)
